go.mod

```diff
@@ -2,7 +2,7 @@ module sigs.k8s.io/cluster-api-provider-aws/v2
 
 go 1.20
 
-replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.4.4
+replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.5.0
```

Comment:
I'm confused by this - what's the effect of replacing a package with the same thing that's required?

Reply:
This will pick the exact version of cluster api specified, otherwise it picks up the dirty version of whatever latest CAPI main branch has.

Reply:
cc @richardcase to correct me if I am wrong.